docs/src/index.md

```diff
@@ -14,21 +14,19 @@ Some presentations on, or using, SumOfSquares:
 The following example shows how to find lower bounds for the Goldstein-Price function using this package.
 
 ```julia
-using MultivariatePolynomials
 using JuMP
-using PolyJuMP
 using SumOfSquares
 using DynamicPolynomials
 using MathOptInterfaceMosek
```

Comment:
MosekTools was just released 3 hours ago deprecating MathOptInterfaceMosek, see https://github.com/JuliaOpt/Mosek.jl/issues/168.
So the new syntax is `using MosekTools` followed by `with_optimizer(Mosek.Optimizer)`.